### DISCUSSION TITLE
react-native-voice Discord!

### DISCUSSION THREAD

**Discussion by safaiyeh** [react-native-voice/voice#283] [General]
Just created a public Discord for everyone to chat! Please use #react-native-voice for voice related conversations.

https://discord.gg/CJHKVeW6sp

**Comment by vum1nhviet** [comment]
Hi, this link is expired,can you update a new link ?

**Reply by amrivera82** [reply]
I this project defunct?

**Comment by safaiyeh** [comment]
This project has been deprecated and archived. Please migrate to [`expo-speech-recognition`](https://github.com/jamsch/expo-speech-recognition). Thank you! ❤️
